### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by rdbreak**
Added the following examples:

- examples/using_blocks.py
- examples/home_tab.py

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,4 +127,4 @@ venv.bak/
 dmypy.json
 
 # Pyre type checker
-.pyre/
+.pyre/
```
Comment (Ambro17) [inline .gitignore:130]: Let's add a newline at the end to keep linters happy
```suggestion
.pyre/

```

File: examples/home_tab.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+from slackify import Slackify, request, reply_text, Slack, ACK
+
+# Initialize App
+app = Flask(__name__)
+slackify = Slackify(app=app)
+cli = Slack(os.getenv('SLACK_BOT_TOKEN'))
+
+
+@slackify.command
+def register():
```
Comment (Ambro17) [inline examples/home_tab.py:12]:
```suggestion
def update_home_view():
```

File: examples/home_tab.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                            }
+                        },
+                    ]
+                }
+            ]
+        }
+    resp = cli.views_publish(
+        user_id=user,
+        view=sample
+        )
```
Comment (Ambro17) [inline examples/home_tab.py:43]:
```suggestion
    )
```

File: examples/home_tab.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    assert resp.get('ok', False), f"Could not build home tab.\nError:\n{resp.data!r}"
+    return reply_text(f'You are `{user}`.')
```
Comment (Ambro17) [inline examples/home_tab.py:45]: Let's change this to `Home tab updated` or something along those lines

File: examples/home_tab.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+
+@slackify.action('tell_joke')
+def tell_a_joke(action):
+    resp = cli.chat_postMessage(
+        channel=action['user']['id'],  # Send the user a private message through slackbot
+        text="I can't remember any joke right now 😅"
+        )
```
Comment (Ambro17) [inline examples/home_tab.py:53]:
```suggestion
    )
```

File: examples/home_tab.py [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    assert resp.get('ok'), f"Error sending message. {resp.data}"
+    return ACK
+
+
+if __name__ == '__main__':
+    print('Running')
```
Comment (Ambro17) [inline examples/home_tab.py:59]: Let's remove this as flask already prints to stdout that the server is running
```suggestion
```

File: examples/home_tab.py [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    app.run('0.0.0.0', port=3000)
```
Comment (Ambro17) [inline examples/home_tab.py:60]: Added newline at the end
```suggestion
    app.run('0.0.0.0', port=3000)

```

File: examples/using_blocks.py [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+cli = Slack(os.getenv('SLACK_BOT_TOKEN'))
+
+
+# Create greeting with formatted text response
+@slackify.message('hello')
+def say_hi(payload):
+    event = payload['event']
+    cli.chat_postMessage(
+        channel=event['channel'],
+        # Blocks can be used to insert markdown support for links, images, and more. Check out https://api.slack.com/reference/block-kit for more.
```
Comment (Ambro17) [inline examples/using_blocks.py:16]: Really nice to include the slack api reference! 👍🏿

File: examples/home_tab.py (hunk 4 above)
Comment (rdbreak) [reply examples/home_tab.py:45]: Are you talking about the file name?

File: examples/home_tab.py (hunk 7 above)
Comment (rdbreak) [reply examples/home_tab.py]: I will add this

**Review by Ambro17 — APPROVED** [review]

**Pull request merged**